ansible/tasks/recreateViews.yml

```diff
@@ -0,0 +1,4 @@
+---
+# Recreates all views in all databases.
+
+- include: db/actionsDbViews.yml
```

Comment:
This is the script, which creates all views in all databases.
Currently we create only views in one Database.
After the seperation of the actions- and activations database, this single script will recreate all views in all transient dbs.